inference-engine/src/multi_device/multi_device_plugin.cpp

```diff
@@ -368,19 +369,32 @@ DeviceInformation MultiDeviceInferencePlugin::SelectDevice(const std::vector<Dev
             if (gpuFullDeviceName.find("iGPU") != std::string::npos) {
                 iGPU.push_back(item);
             } else if (gpuFullDeviceName.find("dGPU") != std::string::npos) {
-                dGPU.push_back(item);
+                // Fixme:  FULL_DEVICE_NAME : Intel(R) Graphics [0x4f87] (dGPU) for DG2 now
+                if (gpuFullDeviceName.find("0x4f87") != std::string::npos) {
```

Comment:
this is clearly a pre-production name :(, maybe we shall delay the support until the device will be more public